src/neuroglancer/mesh/draco/index.ts

```diff
@@ -68,7 +68,7 @@ export async function decodeDracoPartitioned(
   heap.set(buffer, offset);
   numPartitions = partition ? 8 : 1;
   const code = (m.instance.exports.neuroglancer_draco_decode as Function)(
-      offset, buffer.byteLength, partition, vertexQuantizationBits, true);
+      offset, buffer.byteLength, partition, vertexQuantizationBits, false);
```

Comment:
This will need to be an option, since we don't want to change this option for the neuroglancer precomputed datasource.

Reply:
Yes I will do that when the PR is ready to merge. This was just to get the meshes to appear during development.